pyanaconda/core/configuration/payload.py

```diff
@@ -71,6 +89,17 @@ def enable_closest_mirror(self):
         """
         return self._get_option("enable_closest_mirror", bool)
 
+    @property
+    def default_source(self):
+        """Default installation source.
+
+        Valid values:
+
+        CLOSEST_MIRROR
+        CDN
+        """
+        return self._get_option("default_source", SourceType)
```

Comment:
We don't need an enum, but the original value. Look at the validation of the selinux option.

Reply:
I was under the impression we wanted to support all the installation sources but for now document just the two we want to use now, so for that I used the Enum validation as is used for example for partitioning type.

If we want to support only closest mirror and CDN for now, I can indeed use the simplified method as used by SELinux.